scripts/update_siggen.py

```diff
@@ -57,6 +57,7 @@ def main(argv=None):
 
     # Get git sha of source
     source_sha = subprocess.check_output(['git', 'rev-parse', 'HEAD']).strip()
+    source_sha = source_sha.decode('utf-8')
```

Comment:
Whoops--this is a fix for a different thing. It's not a big deal it's here, but it was an accident.